### PR TITLE
Restructure Badges

### PR DESCRIPTION
**Pull request by 9thScientist**
Changes the badges' show

### CODE REVIEW

**Comment by frm** [comment]
I'll wait for #80 since it fixes some of the issues I have with this PR.

File: app/views/badges/show.slim [hunk 1]
```diff
@@ -2,34 +2,25 @@
...
+              = image_tag participant.avatar.url(:thumb), class:'badge-participant-photo-img'
+            td.badge-participant-name
+              a href=edition_participant_path(@edition,participant).sub(/d+\z/, participant.username) = participant.username 
     - else
-      h2
-        | Não existem participantes com este badge
-= javascript_include_tag "share-button", "share.min"
+      .alert
+        .alert-warning
+          h5 Não existem participantes com este badge
```
Comment (dinocosta) [inline app/views/badges/show.slim]: Normally h5 headings are too small, consider using a paragraph instead. If you switch to a paragraph check if the `alert` and `alert-warning` classes can be applied directly to the paragraph :)

File: app/views/badges/show.slim [hunk 2]
```diff
@@ -2,34 +2,25 @@
...
-                  | "> 
-                  = participant.full_name
+      #badge-userlist.row
+        table
+        - @badge.users.each do |participant|
+          tr
+            td.badge-participant
+              = image_tag participant.avatar.url(:thumb), class:'badge-participant-photo-img'
+            td.badge-participant-name
+              a href=edition_participant_path(@edition,participant).sub(/d+\z/, participant.username) = participant.username
```
Comment (dinocosta) [inline app/views/badges/show.slim]: I would suggest you to use the link_to helper :)

File: app/views/badges/show.slim [hunk 3]
```diff
@@ -2,34 +2,25 @@
...
-      = @badge.description
-    .col-md-12
-      = content_tag  :div, "", id: "share-button", data: { description:"Que badge maravilhosa, já viste?", image: "http://intra.seium.org" + @badge.logo_url}
-  #badge-users
+  #badge-logo.col-md-4
+    = image_tag @badge.logo_url, id:"badge-logo-img",  alt:@badge.name
     br
-    - if (users = @badge.users).size > 0
+    = @badge.description
+  #badge-users.col-md-8[style="margin-left: 20px"]
```
Comment (dinocosta) [inline app/views/badges/show.slim]: Could this inline style tag be moved to one of the existing .scss files? If it is needed at all :)

**Review by dinocosta — CHANGES_REQUESTED** [review]

File: app/views/badges/show.slim (hunk 3 above)
Comment (9thScientist) [reply app/views/badges/show.slim]: It's barely needed. But yes, it should be on the scss files. I'm going to change that.

File: app/views/badges/show.slim [hunk 4]
```diff
@@ -6,21 +6,19 @@
...
-            td.badge-participant
-              = image_tag participant.avatar.url(:thumb), class:'badge-participant-photo-img'
-            td.badge-participant-name
-              a href=edition_participant_path(@edition,participant).sub(/d+\z/, participant.username) = participant.username 
+          - @badge.users.each do |participant|
+            tr
+              td.badge-participant
+                = image_tag participant.avatar.url(:thumb), class:'badge-participant-photo-img'
+              td.badge-participant-name
+                = link_to "#{participant.username}",
```
Comment (dinocosta) [inline app/views/badges/show.slim]: Since you are only using `participant.username` you can remove the `"` :)

**Review by dinocosta — CHANGES_REQUESTED** [review]

**Review by dinocosta — APPROVED** [review]

**Pull request merged**